### DISCUSSION TITLE
How to add mechanics notes?

### DISCUSSION THREAD

**Discussion by CoolGamerTheSpy** [BoloVEVO/Kade-Engine#27] [Q&A]
Hey I saw that you answered this question in the kade chat but I was wondering if you could do a tutorial but for this version?

**Comment by BoloVEVO** [comment]
Same steps. For a future update I'll make notetypes a default thing in chart editor.

**Reply by CoolGamerTheSpy** [reply]
The thing is that there I can't do this step because I cant find the code in that I need to replace in Playstate.hx

**Reply by CoolGamerTheSpy** [reply]
![image](https://user-images.githubusercontent.com/75923499/192101064-b5954931-5a9d-405b-a2b2-707bad8be4cf.png)

**Reply by BoloVEVO** [reply]
I forgot I remade some code. Here it is, don't forget that this code is when you miss a note so make your switch statements properly.
![image](https://user-images.githubusercontent.com/88955176/192104727-e4a85a52-c406-480c-9df3-a833ad0ea0ac.png)

**Reply by CoolGamerTheSpy** [reply]
I put the switch statement there right?

**Reply by BoloVEVO** [reply]
Ye

**Reply by CoolGamerTheSpy** [reply]
note kill offest should have a var infort right?

**Reply by CoolGamerTheSpy** [reply]
![image](https://user-images.githubusercontent.com/75923499/192415167-065e39db-92e8-4028-a11a-ca4c04256b8d.png)
